setup.sh

```diff
@@ -1,6 +1,6 @@
 function get-latest {
     curl -s -L https://github.com/hkgumbs/$1/releases/latest \
-        | egrep -o "/hkgumbs/$1/releases/download/v.*\.jar" \
+        | egrep -m 1 -o "/hkgumbs/$1/releases/download/v.*\.jar" \
```

Comment:
This screen scraping is clever and convenient for you as someone developing these projects at the same time, but ultimately it's a better idea to depend explicitly on specific versions. Otherwise any user who has downloaded your `tictactoe-server` code will be broken if they re-fetch dependencies after you make changes to either of them.